ckanext/geodatagov/tests/test_source_form.py

```diff
@@ -0,0 +1,83 @@
+from ckan.plugins import toolkit
+import logging
+from nose.tools import assert_equal, assert_not_in
+import ckanext.harvest.model as harvest_model
+from pylons import config
+
+try:
+    from ckan.tests import helpers, factories
+except ImportError: 
+    from ckan.new_tests import helpers, factories
+
+log = logging.getLogger(__name__)
+
+class TestHarvestSourceForm(helpers.FunctionalTestBase):
+
+    @classmethod
+    def setup_class(cls):
+
+        helpers.reset_db()
+        super(TestHarvestSourceForm, cls).setup_class()
+        harvest_model.setup()
+        sysadmin = factories.Sysadmin()
+        cls.extra_environ = {'REMOTE_USER': sysadmin['name'].encode('ascii')}
+
+    def test_create_waf_collection_harvest_source_form(self):
+
```

Comment:
First off, I think this test class appears to test the functional change in this PR, which is great :-)
This may be a small thing, but I'd prefer to see the "magic" constants (like the URLs on line 34, 38, 50, 51 defined as constants instead of copy/pasted.  E.g.,

```
WAF_HARVEST_SOURCE_URL = 'https://waf-harvest.com/source'
fv['url'] = WAF_HARVEST_SOURCE_URL
fv.submit()
...
fv = res.forms['source-new']
assert_equal(fv['url'], WAF_HARVEST_SOURCE_URL)
```